corehq/apps/hqmedia/models.py

```diff
@@ -940,10 +933,8 @@ def get_media_objects(self, build_profile_id=None):
                 media = CommCareMultimedia.get_doc_class(map_item.media_type)
                 try:
                     media = media.get(map_item.multimedia_id)
-                    retry_successes.append(map_item)
                     yield path, media
                 except ResourceNotFound as e:
-                    retry_failures.append(map_item)
```

Comment:
were these being used so that they show up as variables if the exception is raised in the following line?

Reply:
There used to be a soft assert here. I removed it in https://github.com/dimagi/commcare-hq/commit/d55498ec3c9f453f790c578f73a109e3fecdf810 but missed removing the variables.

Reply:
not sure if same was true for `deleted_media` as well.

Reply:
okay, thanks

Reply:
yeah, `deleted_media` was also used for an email: https://github.com/dimagi/commcare-hq/pull/23871/files#diff-d4f549b022ded4f4837fe295b354b954L551